docs/content/development/modules/create.mdx

```diff
@@ -0,0 +1,344 @@
+---
+description: 'Learn what Modules are and how can you use them during your custom development with Medusa.'
+---
+
+import DocCardList from '@theme/DocCardList';
+import Icons from '@theme/Icon';
+
+# How to Create a Module
+
+In this document, you’ll learn how to create a module, test the module, publish it to NPM, and install it in a Medusa backend.
+
+This document covers the general steps of creating an internal module, but does not explore actually implementing any functionality in it. An internal module is a module that extends or replaces a functionality within the Medusa backend, such as the cache or event bus functionality.
```

Comment:
It is correct. Maybe we could be very specific just to avoid any future confusion with external module. Something like this:

An internal module is a javascript/typescript module that is loaded by Medusa core as part of the  application to extend or replace a functionality within the Medusa backend, such as the cache or event bus functionality.

Reply:
+1